exporter/exporter.go

```diff
@@ -25,6 +25,9 @@ type TraceExporter interface {
 
 	// Name gets the name of the trace exporter.
 	Name() string
+
+	// Shutdown is invoked during service shutdown.
+	Shutdown() error
```

Comment:
I know that this is following the current implementation but I am not sure if there is much value on returning error here, at most just for logging it. That said let's keep as it is.

Reply:
Noted: We can revisit when we look at the Name() interface https://github.com/open-telemetry/opentelemetry-service/issues/265